areas/importer/helsinki.py

```diff
@@ -22,13 +23,22 @@ def import_contract_zones(self):
         logger.info("Helsinki contract zone import done!")
 
     def _fetch_contract_zones(self):
+        contract_zone_filter_str = (
+            ' AND "nimi" NOT IN ({})'.format(
+                ",".join(("'{}'".format(c) for c in EXCLUDED_CONTRACT_ZONES))
```

Comment:
absolutely minimal nitpick: would `for cz in EXCLUDED_CONTRACT_ZONES` make more sense than just `c`? 😅

Reply:
Yes it would! Does it matter at all, probably not! But changed anyway! 😄

Reply:
yeah, not that it matters, but all nitpicks matter 🙌 😅